data/khan/assessmentitems.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
This needs to be removed. Assessmentitems.json is in .gitignore on mainline develop and should not be included.